pkg/cmd/issue/shared/lookup.go

```diff
@@ -13,10 +13,89 @@ import (
 	"github.com/cli/cli/v2/api"
 	fd "github.com/cli/cli/v2/internal/featuredetection"
 	"github.com/cli/cli/v2/internal/ghrepo"
+	o "github.com/cli/cli/v2/pkg/option"
 	"github.com/cli/cli/v2/pkg/set"
 	"golang.org/x/sync/errgroup"
 )
 
+func ParseIssuesFromArgs(args []string) ([]int, o.Option[ghrepo.Interface], error) {
```

Comment:
This is really the same logic as `IssueFromArgWithFields` without actually fetching the PR, and trying to indicate a little bit more with the types what valid states exist.